.github/workflows/analysis.yml

```diff
@@ -66,6 +66,39 @@ jobs:
           echo
           ./scripts/count-bugs.py report/*/index.html
 
+  build_pvs_studio_analyzer:
+    name: PVS-Studio static analyzer
+    runs-on: ubuntu-latest
+    needs: run_linters
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - run:  sudo apt-get update
+      - name: Log environment
+        run:  ./scripts/log-env.sh
+      - name: Install packages
+        run:  |
+          set -xeu
+          wget -nv "https://files.viva64.com/pvs-studio-7.05.35582.25-amd64.deb" -O "pvs.deb" &
```

Comment:
This step seems like a perfect candidate for using cache.

Reply:
Argh; I meant specifically line 83.

Reply:
Great suggestion; will give it a try!